Dockerfile

```diff
@@ -1,145 +1,157 @@
 #--------------------------------------
 # Target image to build
 #--------------------------------------
-ARG IMAGE=latest
+ARG TARGET=latest
 
 #--------------------------------------
 # Ubuntu base image to use
 #--------------------------------------
-ARG FLAVOR=latest
-
-#--------------------------------------
-# renovate rebuild trigger
-#--------------------------------------
-FROM renovate/ubuntu@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-latest
-FROM renovate/ubuntu:bionic@sha256:1ab4cb684944bfd76b2a29c368e4e221b796f08fb826c1a826327476cf196647 as base-bionic
-FROM renovate/ubuntu:focal@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-focal
+# renovate: datasource=docker
+ARG BASE_VERSION=ubuntu:bionic@sha256:ea188fdc5be9b25ca048f1e882b33f1bc763fb976a8a4fea446b38ed0efcbeba
+ARG USER_NAME=ubuntu
```

Comment:
Should we go with "user"?

Reply:
Sorry, I forgot we have `ubuntu` hardcoded in the install scripts for now. Let's parameterize them first before making this configurable